README.md

```diff
@@ -44,5 +44,5 @@ The timers, input and sound are not implemented.
 
 Copyright 2014 Coded Inc. <marc@codedinc.com>
 
-You are free to modify and distribute this however you want. Except for teaching
```

Comment:
Please do not change this notice. I allow anyone to re-use this code (which I use for teaching), except if reusing the code for the sole purpose of teaching how VM works (in a course of book).